### DISCUSSION TITLE
Do the docs on HMR omit the first arg to the accept callback on purpose?

### DISCUSSION THREAD

**Discussion by patreeceeo** [parcel-bundler/parcel#9351] [General]
I noticed that the HRM runtime implements `hot.accept` like so (using TypeScript because it's easier than plain English ha):

```TypeScript
accept(callback: ((getParents: () => Array<Module>) => Array<Module>) | (() => void)): void;
```

This `getParents` function makes it so the HMR runtime in Parcel is actually useful beyond single-file projects, though I just started using it, so maybe too early to say for sure.

Anyway I think Parcel has a lot going for it, and I appreciate all the effort that goes into it. Hope to continue using it and eager to know any more about the HMR implementation/plans.

✌️

**Comment by ChristophP** [comment]
I was wondering the same. I couldn't find anything in the docs about why the HMR callbacks seem to be different from what I've seen from say webpack or so
